mne/evoked.py

```diff
@@ -727,6 +727,42 @@ def plot_white(self, noise_cov, show=True):
         return _plot_evoked_white(self, noise_cov=noise_cov, scalings=None,
                                   rank=None, show=show)
 
+    def animate(self, ch_type, frames=5, interval=100, butterfly=False,
+                blit=True, show=True):
+        """Make animation of evoked data as topomap timeseries.
+
+        Parameters
+        ----------
+        ch_type : str
+            Channel type to plot. Accepted data types: 'mag', 'grad', 'eeg'.
+        frames : int | list of ints
+            If int, the number of frames to animate. If list of ints, the
+            indices to plot in the animation. Defaults to 5.
+        interval : int
+            The time interval before drawing a new frame as milliseconds.
+            Defaults to 100.
```

Comment:
The units of sfreq are Hz so sfreq/10. is also in Hz already. This makes
the animation play out 10x slower than real time regardless of the sample
rate. Any fixed value like 30 or 60 will not exhibit this behavior, and the
rate at which the data actually plays out (data time compared to viewing
time) will vary as a function of sample rate, which is not so nice. Make
sense?
On Dec 15, 2015 2:40 AM, "Alexandre Gramfort" <notifications@github.com>
wrote:

> In mne/evoked.py
> <https://github.com/mne-tools/mne-python/pull/2680#discussion_r47605154>:
>
> > @@ -727,6 +727,42 @@ def plot_white(self, noise_cov, show=True):
> >          return _plot_evoked_white(self, noise_cov=noise_cov, scalings=None,
> >                                    rank=None, show=show)
> >
> > +    def animate(self, ch_type, frames=5, interval=100, butterfly=False,
> > +                blit=True, show=True):
> > +        """Make animation of evoked data as topomap timeseries.
> > +
> > +        Parameters
> > +        ----------
> > +        ch_type : str
> > +            Channel type to plot. Accepted data types: 'mag', 'grad', 'eeg'.
> > +        frames : int | list of ints
> > +            If int, the number of frames to animate. If list of ints, the
> > +            indices to plot in the animation. Defaults to 5.
> > +        interval : int
> > +            The time interval before drawing a new frame as milliseconds.
> > +            Defaults to 100.
>
> +1 for frame_rate in hz
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2680/files#r47605154>.
>